### ISSUE TITLE
Internationalize ligature processor

### ISSUE THREAD

**Issue by DaveJarvis** [DaveJarvis/keenwrite#81]
After #61 is complete, update the ligature processor to be I18N-friendly. In German, for example, this could mean having to differentiate between short s, long s.

It may be necessary to add a little more intelligence to remove certain ligatures from processing. See the following for details: https://english.stackexchange.com/q/50660/22099

**Comment by DaveJarvis** [comment]
Won't fix unless specifically requested.
